steamcmd_servers/dont_starve/egg-don-t-starve-together.json

```diff
@@ -3,12 +3,13 @@
     "meta": {
         "version": "PTDL_v1"
     },
-    "exported_at": "2020-07-10T10:48:40-04:00",
+    "exported_at": "2020-12-25T14:03:44-05:00",
     "name": "Don't Starve Together",
     "author": "parker@parkervcp.com",
     "description": "Don\u2019t Starve Together is an uncompromising wilderness survival game full of science and magic.",
+    "features": null,
     "image": "quay.io\/parkervcp\/pterodactyl-images:debian_source",
-    "startup": "cd bin && coproc caves ( .\/dontstarve_dedicated_server_nullrenderer -console -persistent_storage_root \/home\/container\/DoNotStarveTogether -conf_dir config -cluster server -players {{MAX_PLAYERS}} -shard Caves ); .\/dontstarve_dedicated_server_nullrenderer -bind_ip 0.0.0.0 -port 10999 -console -persistent_storage_root \/home\/container\/DoNotStarveTogether -conf_dir config -cluster server -players {{MAX_PLAYERS}} -shard Master && echo 'c_shutdown()' >&\"${caves[1]}\"",
+    "startup": "cp -f DoNotStarveTogether\/config\/server\/Master\/modoverrides.lua DoNotStarveTogether\/config\/server\/Caves\/modoverrides.lua && cd bin && coproc caves ( .\/dontstarve_dedicated_server_nullrenderer -console -persistent_storage_root \/home\/container\/DoNotStarveTogether -conf_dir config -cluster server -players {{MAX_PLAYERS}} -shard Caves ); .\/dontstarve_dedicated_server_nullrenderer -bind_ip 0.0.0.0 -port 10999 -console -persistent_storage_root \/home\/container\/DoNotStarveTogether -conf_dir config -cluster server -players {{MAX_PLAYERS}} -shard Master && echo 'c_shutdown()' >&\"${caves[1]}\"",
```

Comment:
Instead of copying this on every start why not make it a symbolic link during the install?

Reply:
Because unfortunately Don't starve together fails to read it if its symlinked.

Reply:
I will play with that to see. It shouldn't matter if it's a symlink or not as long as it's relative.